regolith/cvbuilder.py

```diff
@@ -1,47 +1,35 @@
 """Builder for CVs."""
 import os
-import shutil
 import subprocess
 from glob import glob
-from copy import deepcopy
 from itertools import groupby
 
-from jinja2 import Environment, FileSystemLoader
+from regolith.builder_base import BuilderBase
+
 try:
     from bibtexparser.bwriter import BibTexWriter
     from bibtexparser.bibdatabase import BibDatabase
+
     HAVE_BIBTEX_PARSER = True
 except ImportError:
     HAVE_BIBTEX_PARSER = False
 
-from regolith.tools import all_docs_from_collection, date_to_float, \
-    date_to_rfc822, rfc822now, gets, month_and_year
+from regolith.tools import all_docs_from_collection, date_to_rfc822, rfc822now, gets, month_and_year, \
+    filter_publications, \
+    filter_projects, filter_grants, awards_grants_honors, latex_safe, LATEX_OPTS, make_bibtex_file
 from regolith.sorters import doc_date_key, ene_date_key, category_val, \
-    level_val, id_key, date_key, position_key
-
-LATEX_OPTS = ['-halt-on-error', '-file-line-error']
-
-def latex_safe(s):
-    return s.replace('&', '\&').replace('$', '\$').replace('#', '\#')
+    level_val, date_key, position_key
 
-class CVBuilder(object):
 
+class CVBuilder(BuilderBase):
     btype = 'cv'
 
     def __init__(self, rc):
-        self.rc = rc
-        self.bldir = os.path.join(rc.builddir, self.btype)
-        self.env = Environment(loader=FileSystemLoader([
-                    'templates',
-                    os.path.join(os.path.dirname(__file__), 'templates'),
-                    ]))
-        self.construct_global_ctx()
-        if HAVE_BIBTEX_PARSER:
-            self.bibdb = BibDatabase()
-            self.bibwriter = BibTexWriter()
+        super().__init__(rc)
+        self.cmds = ['latex', 'pdf', 'clean']
 
     def construct_global_ctx(self):
-        self.gtx = gtx = {}
+        gtx = self.gtx
```

Comment:
Some of these basic global context constructions (len, true, false, etc) can be moved to the base class